.github/workflows/nightly-ci.yml

```
@@ -120,6 +120,7 @@ jobs:
       target_tag_plain: ${{ needs.vllm-build.outputs.target_tag_plain }}
       cuda_version: '["12.9", "13.0"]'
       platform: '["amd64", "arm64"]' # arm64 for CPU tests, single GPU tests are skipped
+      enable_coverage: true
```

actionlint (1.7.12)

[error] input "enable_coverage" is not defined in "./.github/workflows/shared-test.yml" reusable workflow. defined inputs are "amd_runner", "cpu_only_test_markers", "cpu_only_test_timeout_minutes", "cpu_parallel_mode", "cuda_version", "gpu_test_markers", "gpu_test_timeout_minutes", "platform", "run_cpu_only_tests", "run_gpu_tests", "run_sanity_check", "target_tag_plain", "test_suite_name", "test_type"

(workflow-call)